packages/core/src/controllers/ApiController.ts

```diff
@@ -267,7 +267,7 @@ export const ApiController = {
       ApiController.fetchNetworkImages(),
       ApiController.fetchConnectorImages()
     ]
-    if (OptionsController.state.enableAnalytics === undefined) {
+    if (OptionsController.state.enableAnalytics) {
```

Comment:
Same here, we unles they set it to false, we could enable it like the following
```suggestion
    if (OptionsController.state.enableAnalytics !== false) {
```

Reply:
It is the same idea from before, leaving `undefined` as a positive value is miss leading and carries implicit logic. Are you sure we should change to that?